mmpy_bot/plugins/base.py

```diff
@@ -88,21 +70,213 @@ async def call_function(
             # a plugin-specific thread or process pool if we wanted.
             self.driver.threadpool.add_task(function, event, *groups)
 
-    def get_help_string(self):
-        string = f"Plugin {self.__class__.__name__} has the following functions:\n"
-        string += "----\n"
-        for functions in self.message_listeners.values():
-            for function in functions:
-                string += f"- {function.get_help_string()}"
-            string += "----\n"
-        if len(self.webhook_listeners) > 0:
-            string += "### Registered webhooks:\n"
-            for functions in self.webhook_listeners.values():
-                for function in functions:
-                    string += f"- {function.get_help_string()}"
+
+class HelpPlugin(Plugin):
```

Comment:
Can you move this Plugin to its own file? `ExamplePlugin` has its own file too, and this one is getting quite large.